README.md

```diff
@@ -54,7 +48,7 @@ You can build the project using the command:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/fasaldana/Leaderboard/issues).
 
 ## Show your support
 
```

Comment:
- [ ] The `MIT` link is broken, Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.